### DISCUSSION TITLE
The erb action should use ErbTemplateHelper and support trim_mode

### DISCUSSION THREAD

**Discussion by jschnare** [fastlane/fastlane#16867] [Ideas] [answered]
PR #11144 adds the 'trim_mode' parameter to ErbTemplateHelper to control how blank lines are handled.

The erb action should also accept a 'trim_mode' parameter.  It's also probably a good idea to to use ErbTemplateHelper instead of calling ERB directly.

**Comment by joshdholtz** [comment, marked as answer]
OMG, I didn’t know trim mode was a thing 😱 I actually needed this a few weeks ago.

But also, this makes sense to me! I’ll create an issue to get this added with the “complexity: you-can-do-this” label on it. I have a few other things I’m working on at the moment but hopefully somebody else can pick that up before I can get to it 😊

**Reply by joshdholtz** [reply]
@jschnare Issue created 👉 https://github.com/fastlane/fastlane/issues/16869

**Reply by joshdholtz** [reply]
@jschnare PR is here 👉 https://github.com/fastlane/fastlane/pull/16882

I actually needed this today 😊
